debug:
  verbose: True
  visualize: True
  save_images: True

camera:
  info_topic: '/camera/color/camera_info'
  image_topic: '/camera/color/image_raw'
  depth_topic: '/camera/depth/image_rect_raw'

model:
  model: Mask_RCNN
  path: '/home/amiga/nimo_ws/src/NiMo-Perception_v2/models/model_field_day1.pth'
  score_threshold: 0.6
  device: -1

stalk:
  minimum_mask_area: 30
  feature_point_offset: 10